src/components/BaseModal/BaseModal.jsx

```diff
@@ -34,15 +34,16 @@ export const BaseModal = ({setModalVisibility, children}) => {
     }, []);
 
     return (
-        <ModalBackground top={topOffset}>
-            <ModalContainer>
-                <ModalContainerCloseBtnWrapper onClick={() => closeModalHandler()}>
-                    <CloseBtnImg src={closeCross}>
-                    </CloseBtnImg>
-                </ModalContainerCloseBtnWrapper>
-                {children}
-            </ModalContainer>
-        </ModalBackground>
+        modalVisibility &&
```

Comment:
For what case do you need this flag? As I see its possible to use modal without it. 
In parent you could use 
```
const [modalVisibility, setModalVisibility] = useState();
...
modalVisibility && <BaseModal setModalVisibility={setModalVisibility}/>
```